index.html

```diff
@@ -135,6 +143,86 @@ <h2>Project name goes here</h2>
       </div>
     </section>
 
+    <section class="desktop-work">
+      <h2 class="desktop-projects">Projects</h2>
+      <div class="my-first-project">
+        <div class="project-overlay"></div>
+        <h2>Project name goes here</h2>
+        <ul>
+          <li>HTML/CSS</li>
+          <li>Ruby on Rails</li>
+          <li>JavaScript</li>
+        </ul>
+        <button type="button">
+          See this project
+          <img src="assets/desktop-version/Union.png" alt="right arrow icon" />
+        </button>
+      </div>
+      <div class="my-second-project">
+        <h2>Project name goes...</h2>
+        <ul>
+          <li>HTML/CSS</li>
+          <li>Ruby on Rails</li>
+          <li>+1</li>
+        </ul>
+        <button type="button">
+          See this project
+          <img src="assets/desktop-version/Union.png" alt="right arrow icon" />
+        </button>
+        <div class="overlay-2"></div>
+      </div>
+    </section>
+
+    <section class="desktop-about">
+      <div class="technologies-languages">
+        <h2>Languages</h2>
+        <ul>
+          <li>JavaScript (ES6+)</li>
+          <li>Ruby</li>
+          <li>HTML</li>
+          <li>CSS</li>
+        </ul>
+        <div class="hashtag-desktop"></div>
+      </div>
+      <div class="technologies-framework">
+        <h2>Frameworks</h2>
+        <ul>
+          <li>React.js</li>
+          <li>Ruby on Rails</li>
+          <li>RSpec</li>
+          <li>Capibara</li>
+        </ul>
+        <div class="framework-desktop"></div>
+      </div>
+      <div class="technologies-skills">
+        <h2>Skills</h2>
+        <ul>
+          <li>Database Management</li>
+          <li>CLI</li>
+          <li>API Design</li>
+          <li>Version Control</li>
+          <li>Web Development</li>
+        </ul>
+        <div class="skills-desktop"></div>
+      </div>
+      <div class="grid">
+        <div class="images"></div>
+      </div>
+    </section>
+
+    <aside class="about-me-desktop">
+      <h2>About</h2>
+      <p>
+        I can help you build a product , feature or website Look through <br />
+        some of my work and experience! If you like what you see and <br />
```

Comment:
- [OPTIONAL]  This is marked optional though, but it is highly recommended to attend to it so that your design can match your chosen template so that you will have something fantastic presented to your potential recruiters. Take a closer look at both your design and the figma design and walk your design alongside the Figma design.

### Your Design
![your-about](https://user-images.githubusercontent.com/26597432/236647886-c9c5717c-1585-4b83-925c-7c10031fb32b.png)

### Figma Design
![tem-about](https://user-images.githubusercontent.com/26597432/236647890-0fa3b700-9df0-42d0-acc1-1089f4f20d2b.png)